.github/workflows/macos.yml

```diff
@@ -4,7 +4,7 @@ on:
     branches:
       - develop
       - master
-  pull_request:
+  pull_request_target:
```

Comment:
https://github.community/t/pull-request-target-not-triggering-workflow/163924/6

I think in order for this to run the actions, it has to been "develop" before it will run the workflow.

Reply:
Can you elaborate on this statement? I don't really understand your comment or the linked comment.